server/routerlicious/packages/lambdas/src/nexus/interfaces.ts

```diff
@@ -18,6 +17,7 @@ import type {
 	IThrottler,
 	IWebSocketTracker,
 } from "@fluidframework/server-services-core";
+import { IEvent } from "../events";
```

Comment:
This interface had already been copied to server/routerlicious in the past and the local copy is being in several files, we just missed changing the import in this file.